### ISSUE TITLE
Accessible Color Contrast

### ISSUE THREAD

**Issue by andrewwoods** [andrewwoods/opal#26]
## Description

By default, terminals use a set of 16 basic colors - 8 normal, 8 bold. It can be difficult to create a set of color combinations that are readble. For example Normal Red on a Black background has very low contrast.

Most modern terminal applications e.g. iTerm2 allow you to create pallettes of color presets that let you define how the ANSI standard colors should be rendered. Transparency and background blurring also impact contrast.

There are also pre-made terminal themes. I'd like to create a list of the existing themes that are accessible. This is to help people remove the guesswork. Ideally it'd be great to create an office Opal Dark and Opal Light terminal themes. These themes aren't required for the release of version 3.0, but probably sometime in the 3.x series.

## Expected Outcome

It should be easy to create readable colored text. I'd also like to figure out how to calculate the WCAG color contrast ratios. So I can show the user.

I'd also likt to provide either a list of 

**Describe alternatives you've considered**

1.   256color2.pl
2.   ansi --colortable
3.   ANSI_test.sh
4.   palette_test.py from Urwid examples
5.   terminal-colors
6.   xterm_colour_chart

These are existing scripts that will generate color combinations in the terminal. I think `ansi` is the best option I've seen so far, and you can examine the [ansi project on Github](https://github.com/fidian/ansi).
